scripts/utils/termux_reuse_pr_build_artifacts.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+set -euo pipefail
+
+# Check if we can perform CI fast path
+# Here we make sure all commits come from single PR, make sure nor buildsystem nor dependencies were not changed
+# and if so we are reusing PR check artifacts and uploading them to apt repo to make save some CI time
+
+
+# Make sure we are running in a push triggered by PR merge.
+[[ "${GITHUB_EVENT_NAME:-}" == "push" ]] || exit 0
+
+error_exit() {
+  echo "$@"
+  exit 0
+}
```

Comment:
`error_exit` is very close to `termux_error_exit` which we do not wanna create confusion with.

Consider either using a more generic "`die`" function, e.g.
I think you can probably also bake in the `, not performing CI fast path` into the function since you added it to every call of this error function.
```bash
die() {
    echo "$*, not performing CI fast path"
    exit 0
    # I think this should be an exit 1
    # that way we can handle fast path failure in the workflow with a 
    # `./scripts/utils/termux_reuse_pr_build_artifacts.sh || echo "Nope."`
} >&2
```
Also make sure it prints to stderr.

Reply:
This function is needed to print some reasoning why we return early here. None of these "errors" is critical to workflow and this script must not cause CI fail. 
I will rename it to `infoexit`.